tests/test_wave_01.py

```diff
@@ -60,13 +60,15 @@ def test_get_task_not_found(client):
     # Assert
     assert response.status_code == 404
 
-    raise Exception("Complete test with assertion about response body")
+    # raise Exception("Complete test with assertion about response body")
     # *****************************************************************
     # **Complete test with assertion about response body***************
     # *****************************************************************
```

Comment:
Likewise, you should feel free to get rid of these scaffolding comments as well as now-unnecessary code like the `raise` statement.